### PR TITLE
[MNG-7661] Replace deprecated 'verifier.addCliOption()'

### PR DESCRIPTION
**Pull request by psiroky**
This is another clean-up PR - slightly smaller than the the last one, but still pretty big (even though 95% of the changes were again done automatically with find&replace).

@slawekjaranowski thanks for merging the other PRs!

### CODE REVIEW

**Comment by gnodet** [comment]
@psiroky could you fix the conflict please ?

File: core-it-suite/src/test/java/org/apache/maven/it/MavenITmng4412OfflineModeInPluginTest.java [hunk 1]
```diff
@@ -93,10 +93,10 @@ public void testitCollector()
...
         verifier.deleteArtifacts( "org.apache.maven.its.mng4412" );
         verifier.filterFile( "settings-template.xml", "settings.xml", "UTF-8", verifier.newDefaultFilterProperties() );
-        verifier.addCliOption( "-Pcollector" );
-        verifier.addCliOption( "--offline" );
-        verifier.addCliOption( "-s" );
-        verifier.addCliOption( "settings.xml" );
+        verifier.addCliArgument( "-Pcollector" );
+        verifier.addCliArgument( "--offline" );
+        verifier.addCliArgument( "-s" );
+        verifier.addCliArgument( "settings.xml" );
```
Comment (psiroky) [inline core-it-suite/src/test/java/org/apache/maven/it/MavenITmng4412OfflineModeInPluginTest.java:99]: These successive calls to `addCliArgument` could be possibly replaced by single call to
```
addCliArguments("-Pcollector", "--offline", "-s", "settings.xml");
```
I am not sure though if that's making things easier or harder to read (it may be just matter of preference). In any case, those would be done in different PR anyway.

**Review by cstamas — APPROVED** [review]

**Comment by cstamas** [comment]
@psiroky sorry for causing inconvenience, and I am aware this is 2nd time, but please can you fix the conflict? :smile: and then let's merge @slawekjaranowski

**Comment by psiroky** [comment]
@cstamas no worries, the conflict was trivial to resolve. Should be ready now.

**Comment by slawekjaranowski** [comment]
First I want to merge: https://github.com/apache/maven/pull/953 to be sure that tests are ok.
